pavelib/paver_tests/test_assets.py

```diff
@@ -0,0 +1,56 @@
+"""Unit tests for the Paver asset tasks."""
+
+import ddt
+from paver.easy import call_task
+
+from .utils import PaverTestCase
+
+
+@ddt.ddt
+class TestPaverAssetTasks(PaverTestCase):
+    """
+    Test the Paver asset tasks.
+    """
+    @ddt.data(
+        [""],
+        ["--force"],
+        ["--debug"],
+        ["--system=lms"],
+        ["--system=lms --force"],
+        ["--system=studio"],
+        ["--system=studio --force"],
```

Comment:
if I'm reading the code right, you can also pass in `--system=lms,studio`. Is that right? You should add that if so.